<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ngl Qt Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="s_mainGridLayout">
    <item row="0" column="0">
     <spacer name="NGL_spacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>679</width>
        <height>597</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="meshInfoTabGrp">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Import">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QGroupBox" name="s_transformGB">
          <property name="title">
           <string>Mesh Load</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLineEdit" name="m_directoryPath">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>.\models</string>
             </property>
             <property name="maxLength">
              <number>1000</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="m_loadPath">
             <property name="text">
              <string>Dir&lt;&lt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="m_objectSelection"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="m_loadObj">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="s_transformGB_2">
          <property name="title">
           <string>Animation</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QPushButton" name="m_plusFrame">
             <property name="text">
              <string>+Frame</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="m_minusFrame">
             <property name="text">
              <string>-Frame</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="m_toggleAnim">
             <property name="text">
              <string>Toggle</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="s_drawGB">
          <property name="title">
           <string>Update</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QComboBox" name="m_skinType">
             <item>
              <property name="text">
               <string>Linear Blend</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dual Quaternion</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Stretch Twist</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="m_wireframe">
             <property name="text">
              <string>WireFrame</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="m_colour">
             <property name="text">
              <string>Choose Colour</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="m_debugFPS">
             <property name="text">
              <string>Debug</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>212</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
